src/cascadia/TerminalApp/lib/TerminalAppLib.vcxproj

```diff
@@ -263,6 +267,7 @@
     <Midl Include="../CommandKeyChordVisibilityConverter.idl" />
     <Midl Include="../Tab.idl" />
     <Midl Include="../TerminalSettings.idl" />
+    <Midl Include="..\ColorScheme.idl" />

```

Comment:
`/`

Reply:
haha. I'm gonna be hitting this _a lot_ because VS adds it in like this. I'll keep an eye out for it :P